java-lib/src/main/java/org/flinkextended/clink/feature/onehotencoder/ClinkOneHotEncoderModel.java

```diff
@@ -43,42 +44,59 @@
 import org.apache.flink.util.Preconditions;
 
 import com.sun.jna.Pointer;
+import org.apache.commons.compress.utils.Lists;
 import org.apache.commons.lang3.ArrayUtils;
+import org.clink.feature.onehotencoder.OneHotEncoderModelDataProto;
 import org.flinkextended.clink.jna.ClinkJna;
 import org.flinkextended.clink.jna.SparseVectorJna;
 import org.flinkextended.clink.util.ByteArrayDecoder;
 import org.flinkextended.clink.util.ByteArrayEncoder;
-import org.flinkextended.clink.util.JnaUtils;
-import org.flinkextended.clink.util.ParamUtils;
+import org.flinkextended.clink.util.ClinkReadWriteUtils;
 
 import java.io.IOException;
 import java.util.*;
-import java.util.function.Function;
-
-import static org.apache.flink.ml.util.ParamUtils.initializeMapWithDefaultValues;
+import java.util.stream.Collectors;
 
 /**
  * Wrapper class for Flink ML OneHotEncoderModel which calls equivalent C++ operator to transform.
  */
 public class ClinkOneHotEncoderModel
         implements Model<ClinkOneHotEncoderModel>, OneHotEncoderParams<ClinkOneHotEncoderModel> {
     private final Map<Param<?>, Object> paramMap = new HashMap<>();
-    private Table modelDataTable;
+    private String modelDataPath;
+    private transient Table modelDataTable;
```

Comment:
It makes sense to skip the transient for now as it only makes slight influence to this PR, that `ClinkOneHotEncoderModel` must use static classes for inner functions and operators, instead of making them member class or anonymous class. I'll remove the `transient` key word here.